packages/mds-policy/schema-gen/PermittedVehiclePolicySchema.json

```diff
@@ -0,0 +1,268 @@
+            "nullable": false,
+            "minItems": 1
+          },
+          "maximum": {
+            "type": "number",
+            "nullable": true
+          },
+          "messages": {
+            "type": "object",
+            "required": [],
+            "nullable": true,
+            "default": null
+          },
+          "minimum": {
+            "type": "number",
+            "nullable": true
+          },
+          "modality": {
+            "type": "string",
+            "pattern": "micromobility",
+            "nullable": true
+          },
+          "name": {
+            "type": "string"
+          },
+          "rule_id": {
+            "type": "string",
+            "format": "uuid"
+          },
+          "rule_type": {
+            "type": "string",
+            "pattern": "count"
+          },
+          "rule_units": {
+            "type": "string",
+            "nullable": true,
+            "default": null
+          },
+          "rate_amount": {
+            "type": "number",
+            "nullable": true,
+            "default": null
+          },
+          "rate_recurrence": {
+            "type": "string",
+            "enum": [
+              null,
+              "once",
+              "each_time_unit",
+              "per_complete_time_unit"
+            ],
+            "nullable": true,
+            "default": null
+          },
+          "states": {
+            "type": "object",
+            "properties": {}
+          },
+          "start_time": {
+            "type": "string",
+            "nullable": true,
+            "pattern": "^\\d{2}:\\d{2}:\\d{2}$"
+          },
+          "value_url": {
+            "type": "string",
+            "nullable": true
+          },
+          "vehicle_types": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "car",
+                "bicycle",
+                "scooter",
+                "moped",
+                "other"
+              ]
+            },
+            "nullable": true
+          },
+          "service_types": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "standard",
+                "shared",
+                "luxury"
+              ]
+            },
+            "nullable": true,
+            "default": null
+          },
+          "transaction_types": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "pick_up",
+                "drop_off"
+              ]
+            },
+            "nullable": true,
+            "default": null
+          },
+          "propulsion_types": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "human",
+                "electric",
+                "electric_assist",
+                "hybrid",
```

Comment:
We allow hybrid as a propulsion type here?

Reply:
`PROPULSION_TYPES` from `@mds-core/mds-types` yeah, that's the import